### PR TITLE
feat: derive BorshSerialize for reference

### PR DESCRIPTION
**Pull request by bowenwang1996**
Implement `BorshSerialize` for `&T` if `T` implements `BorshSerialize`.

### CODE REVIEW

**Comment by frol** [comment]
@chefsale Do know why Travis CI hasn't started for this PR?

**Comment by chefsale** [comment]
I do not see an travis integration here at all, am I missing something or we don't have travis setup for this repo at all? @frol

**Comment by frol** [comment]
@chefsale What do you mean? https://github.com/near/borsh/blob/master/.travis.yml

**Comment by chefsale** [comment]
But doesn't travis has to be setup in the integrations here as well: https://github.com/near/borsh/settings/installations i have seen that it got triggered a while ago on the other PR: https://github.com/near/borsh/pull/70

**Comment by chefsale** [comment]
https://travis-ci.com/github/near/borsh

**Comment by chefsale** [comment]
Yes, the repo was deactivated in travis.

File: borsh-rs/borsh/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "borsh"
-version = "0.7.0"
+version = "0.8.0"
```
Comment (frol) [inline borsh-rs/borsh/Cargo.toml:3]: Why do you update the minor version? According to semver, releases before 1.0 treated differently and minor version change indicates a breaking change. In this case, it is a new feature, so we can just bump the patch version. /cc @chefsale

Comment (bowenwang1996) [reply borsh-rs/borsh/Cargo.toml:3]: I see.

Comment (bowenwang1996) [reply borsh-rs/borsh/Cargo.toml]: Fixed

**Pull request merged**